css/main.css

```diff
@@ -1,3 +1,112 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+.main {
+  width: 100%;
+  text-align: center;
+  background: var(--gray);
+}
+.hero {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  background: url(../img/hero.webp);
+  color: var(--white);
+  align-items: center;
+  padding: 4em 2em;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.hero_header_logo {
+  margin: auto 10px;
+}
+.hero_header_logo img {
+  width: 100%;
+  max-width: 100%;
+}
+.hero_header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+.hero h1 {
+  font-size: 8vw;
```

Comment:
Using `vw` for your font sizes is resulting in your fonts being tiny for mobile, and too big for desktop. It's best to use fixed font sizes (using `rem` or `px`) and change this size in your media queries to give different values for different breakpoints.